react/features/app-settings/components/styles.js

```diff
@@ -23,6 +23,7 @@ export default createStyleSheet({
      */
     backIcon: {
         alignSelf: 'center',
+        paddingRight: 12,
```

Comment:
@zbettenbuk, could we please have the back button bigger as well? It's too small to our liking on an actual iPhone 7 Plus and presents a difficult hit target.

Reply:
Good point, thanks! Too large looks bad too though, but I try to go with something nice.

Reply:
GitHub doesn't mark it outdated because the change was in another line

Reply:
I tested 22 here and feels better, can you please change it?